bin/setup

```diff
@@ -10,13 +10,13 @@ set -e
 gem install bundler --conservative
 bundle check || bundle install
 
-# Set up configurable environment variables
-if [ ! -f .env ]; then
-  cp .sample.env .env
-fi
-
 # Set up database and add any development seed data
-bundle exec rake db:setup dev:prime
+bundle exec rake db:setup
```

Comment:
That "dev:prime" was one of the reason to the problems with Heroku's review apps since it only works on development environment.